engines/docker/sandbox.go

```diff
@@ -277,5 +298,9 @@ func (s *sandbox) dispose() error {
 	if hasErr {
 		return runtime.ErrNonFatalInternalError
 	}
+
+	if s.videoDev != nil {
+		s.videoDev.claimed = false
```

Comment:
It's probably best to decide what is ecapsulated where... You have a method `videoDeviceManager.release`